docs/UserGuide.md

```diff
@@ -157,16 +175,17 @@ Examples:
 
 ### Canceling an appointment : `cancel`
 
-Delete the specified appointment from the appointment book.
+Deletes the specified appointment at the date and time indicated from the appointment book.
 
-Format `cancel APPT_ID`
+Format `cancel d/DATE t/TIME`
 
-* Deletes the appointment at the specified `APPT_ID`.
-* The `APPT_ID` must be a positive integer 1, 2, 3, …​
-* The `APPT_ID` is a unique id containing information on the appointment date and time.
+* Deletes the appointment at the specified `DATE and TIME`.
+* The `DATE and TIME` indicated must take place in the future.
+* An appointment with the corresponding `DATE and TIME` must exist in the appointment book.
```

Comment:
Personally, I think it'll be a good addition to have this enhancement for our future iterations as it may seem like a bug if we are able to `cancel` an appointment that happened in the past. However, I realized that it shouldn't be the case for `done` since we will only mark an appointment as completed after it has occurred. Thanks for pointing it out Jin Hao :)

Reply:
I would think that we should disable `cancel` if the date of the respective appointment to be cancelled is alright passed since there is no point to cancel an appointment which is already in the past. 
However, I think it would be different for `done` since there may be cases that the admin forgets to mark done and just lets it expire. `done` allows for the correction.

How about the rest?

Reply:
Yup I agree with you too Wanlin.